lib/praxis/plugins/praxis_mapper_plugin.rb

```diff
@@ -103,7 +105,7 @@ def setup!
           unless log_stats == 'skip'
             Praxis::Notifications.subscribe 'praxis.request.all' do |name, *junk, payload|
               if (identity_map = payload[:request].identity_map)
-                PraxisMapperPlugin::Statistics.log(identity_map, log_stats)
+                PraxisMapperPlugin::Statistics.log(payload[:request], identity_map, log_stats)
```

Comment:
it's a best practice and testing reason. If we shortcut it outside, somebody might call it without checking the request flag. By doing it this way, there's no way to bypass the requirement (unless you're doing it on purpose)